### DISCUSSION TITLE
Does kubescape have any restrictions on the k8s version?

### DISCUSSION THREAD

**Discussion by testwill** [kubescape/kubescape#972] [Q&A] [answered]
Are there any restrictions on version

**Comment by dwertent** [comment]
Not that I know of.
I believe that older Kubernetes versions might not be supported.

**Reply by testwill** [reply]
thanks

**Comment by craigbox** [comment, marked as answer]
We have some controls that aren’t relevant in certain versions. For example, there are controls that run on `PodSecurityPolicies`, which are deprecated from a certain Kubernetes version, but nothing will break - they will appear as passing/irrelevant.
